docs/preview/features/message-pumps/service-bus.md

```diff
@@ -185,6 +185,10 @@ public class Startup
                 // (default: Transaction-Id).
                 options.Correlation.TransactionIdPropertyName = "X-Transaction-ID";
 
+                // Indicate whether or not the default built-in JSON deserialization should ignore missing members 
+                // when deserializing the incoming message (default: false).
+                options.Deserialization.IgnoreMissingMembers = true;
```

Comment:
I think it's a different perspective but what about this?
```suggestion
                options.Deserialization.IgnoreAdditionalFields = true;
```

This feels a bit more clear that it's aiming towards additional info in the payload. What do you think @fgheysels ?

Reply:
We could create some email for this too. I created this one but seems like an enum is maybe better as other things could/should be configurable for consumers.

Reply:
What do you mean with email? Or did you mean enum?

Enum is fine for me as well.

Reply:
Yeah, enum 😅